src/components/TaskInput.jsx

```diff
@@ -0,0 +1,26 @@
+import { useContext } from "react";
+import ToDoContext from "../context/ToDoContext";
+import { AiFillPlusCircle } from "react-icons/ai";
+const TaskInput = () => {
+  const { toDo, handleOnChange, task, handleTaskAdd } = useContext(ToDoContext);
+  console.log(toDo);
+  return (
+    <div className="bg-teal-50 flex justify-between items-center space-x-3 px-4 w-[50%] rounded-full border shadow-3xl">
+      <input
+        type="text"
+        name="taskName"
```

Comment:
This attribute is not really important unless you are handling your form in the object state and your state is a string state.